vagrant/hyperglass-server.pp

```diff
@@ -0,0 +1,14 @@
+# If you enable selinux, redis will not be able to write to its data directory
+# and will not properly start without the following.
+selinux::permissive { 'redis_t':
```

Comment:
do we still need this? I think the redis module sets correct permissions?

Reply:
Yeah :(

Once I fixed this, then redis worked properly.